src/pages/Environments.js

```diff
@@ -334,13 +334,12 @@ const Environments = () => {
             }
           },
           {
-            size: { basis: 90, grow: 0 },
+            size: { basis: 100, grow: 0 },
             headerRenderer: () => h(Sortable, { sort, field: 'created', onSort: setSort }, ['Type']),
             cellRenderer: ({ rowIndex }) => {
               const cloudEnvironment = filteredCloudEnvironments[rowIndex]
-              // TODO: update return logic once we support more app types (will need a backend change to return appType in list apps endpoint as well)
               return cloudEnvironment.appName ?
-                'Galaxy' :
+                (cloudEnvironment.appType ? _.capitalize(cloudEnvironment.appType) : 'Galaxy') :
```

Comment:
I think we should probably change this to a `Utils.cond` at this point for simplicity

Reply:
I hope this will eventually get down to just

`return _.capitalize(cloudEnvironment.appName ? cloudEnvironment.appType : cloudEnvironment.runtimeConfig.cloudService)`

Reply:
I like the isApp idea, but I'm not sold on the appTypes const yet because my hope is that we can get rid of any need to have the app names listed in terra-ui at all. I may be overly naive about that, but given that my goal with these changes is to get rid of "Galaxy" in these spots, it seems strange to temporarily introduce an appTypes const.

Reply:
We use `'Galaxy'` in a lot of places outside this file too and with `appTypes` I was hoping we could slowly get rid of those as well.

Reply:
@kyuksel I promise that the first time I find myself putting "Cromwell" in some of the terra-ui code (in reference to an App) I will create appTypes.

Reply:
👍🏼